utils/jenkins/Jenkinsfile

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2019-2020, Intel Corporation */
+
+/* Jenkinsfile - Scripts to create pmemkv and pmemkv_matrix jobs - to run with initial jenkins job. */
+
+/* define common properties of pipelines: */
+description_param = '''
+	string {
+		name('DESCRIPTION')
+		defaultValue('')
+		trim(true)
+		description('description')
+	}
+'''
+
+label_param = '''
+	string {
+		name('LABEL')
+		defaultValue('fedora')
```

Comment:
can we extend the default value to have all our currently supported OSes - I say at least "fedora,ubuntu" ...?